pkg/cephmgr/osd/daemon.go

```diff
@@ -64,13 +72,19 @@ func Run(dcontext *clusterd.DaemonContext, agent *OsdAgent) error {
 		return fmt.Errorf("failed to configure devices. %+v", err)
 	}
 
-	if len(devices.Entries) == 0 {
-		dirs := map[string]int{context.ConfigDir: -1}
-		logger.Infof("configuring osd dir: %+v", dirs)
-		err = agent.configureDirs(context, dirs)
-		if err != nil {
-			return fmt.Errorf("failed to configure dir %s. %+v", context.ConfigDir, err)
-		}
+	// initialize the data directories
+	dirs, err := getDataDirs(dcontext, true)
```

Comment:
we seem to support multiple directories, but i'm not seeing how that would actually be specified. what am i missing?